# Disallow public members within React component classes (react-prefer-private-members)

This rule enforces all non-React-specific members be marked private in React class components.

## Rule Details

When a member of class is marked private, it cannot be accessed from outside of its containing class. This restriction is prefered when writing React components, where encapsulation is almost always desireable.

Examples of **incorrect** code for this rule:

```js

// fill me in

```

Examples of **correct** code for this rule:

```js

// fill me in

```

## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.
